atomic_reactor/plugins/exit_store_metadata_in_osv3.py

```diff
@@ -284,8 +296,10 @@ def run(self):
         # For arrangement version 4 onwards (where group_manifests
         # runs in the orchestrator build), restore the repositories
         # metadata which orchestrate_build adjusted.
-        if PLUGIN_GROUP_MANIFESTS_KEY in self.workflow.postbuild_results:
-            annotations['repositories'] = json.dumps(self.get_repositories())
+        if not self.source_build:
+            if PLUGIN_GROUP_MANIFESTS_KEY in self.workflow.postbuild_results:
+                annotations['repositories'] = json.dumps(self.get_repositories())
+
```

Comment:
This smells like technical debt. Filed OSBS-8332 for it.

Reply:
yeah I don't think it affects anything because we've already assigned that same value above,
and nothing is changing that in this plugin